apps/zbugs/api/index.ts

```diff
@@ -178,22 +178,17 @@ async function mutateHandler(
     throw e;
   }
 
-  const postCommitTasks: (() => Promise<void>)[] = [];
-  const mutators = createServerMutators(jwtData, postCommitTasks);
-
   const response = await handleMutationRequest(
     dbProvider,
     (transact, _mutation) =>
-      transact((tx, name, args) => getMutation(mutators, name)(tx, args)),
+      transact((tx, name, args) =>
+        getMutation(mutators, name)(args)(tx as any, jwtData),
```

Comment:
I do not like this 3 level call.

Let's break it down:

```ts
transact((tx, name, args) => {
  const mutator = getMutator(mutators, name);
  const thunk = mutator(args);
  return thunk(tx, context);
})
```

I think it should be:

```ts
transact((name, args) => {
  const mutator = getMutator(mutators, name);
  return mutator(args);
})
```

so the tx and context needs to be passed somewhere else:

- `tx` comes from the DbProvider. No need to pass that in again.
- `context` gets passed into `handleMutateRequest` (new API name).

```ts
const response = await handleMutateRequest(
  dbProvider,
  (name, args) => {
    const mutator = getMutator(mutators, name);
    return mutator(args);
  },
  request,
  context,
);
```